docs/validation.md

```diff
@@ -10,8 +10,72 @@ in the backend and frontend as of its defined small scope. This allows
 frontends in the browser to validate data in exactly the same way as
 the server will, while not limiting validation to simple regexes.
 
-This speeds up user interaction a good deal
+This speeds up user interaction a good deal. JEXL expressions are used in
+the following contexts:
 
+* Deciding whether a field should be visible or not (`is_hidden` property)
+* Deciding whether a field is required (`is_required` property)
+* ...
```

Comment:
Also:
 - `Task.address_groups`
 - `Flow.next`

But maybe not somethinf to put into `validation.md`.